package/generateScss.mjs

```diff
@@ -129,11 +136,37 @@ const generateSCSSModule = async (moduleName, importObj) => {
   await writeSCSSModule(moduleName, generatedScss);
 };
 
+// Seperate scss module for colors-oklch.scss
+const generateOklchScss = async (importObj) => {
+  let oklchScss = '';
+  const { 'oklch-colors': oklchColors, 'oklch-hues': oklchHues } = importObj;
+
+  for (const [hueKey, hueValue] of Object.entries(oklchHues)) {
+    const hueName = hueKey.replace('--hue-', '');
+
+    for (let i = 0; i <= 15; i++) {
```

Comment:
why not loop over the `OklchColors` object instead of hardcoding the numbers and the `-bright` key? would be cleaner and also would make it future-proof in case the keys change.